### DISCUSSION TITLE
Is it possible to specify the storage class when using AWS S3 as backupstore?

### DISCUSSION THREAD

**Discussion by memaldi** [longhorn/longhorn#2409] [Q&A] [answered]
Hi,

I was wondering if there is a way to specify the storage class when using AWS S3 as backupstore, to upload my backups directly to the infrequent access storage class, without using the standard storage class.

Thank you!

**Comment by joshimoo** [comment, marked as answer]
Currently not, you can use the S3 bucket lifecycle management to transition the objects to your desired storage class.
Also keep in mind that longhorn backups are incremental, so we need to access the `volume.cfg` and previous `backup.cfg`
See:
- https://docs.aws.amazon.com/AmazonS3/latest/userguide/storage-class-intro.html#sc-howtoset
- https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lifecycle-mgmt.html

**Reply by memaldi** [reply]
Thank you!

**Comment by rasmusson** [comment]
Still this would be a nice feature. Currently my backups is lying in standard storage for a while a lifecycle has a time limit before it can be moved. Is this on the radar in future releases?
